openy_data_wrapper/src/DataWrapper.php

```diff
@@ -117,11 +128,13 @@ public function getPins($type, $id = NULL) {
       $location_ids[] = $id;
     }
     else {
+      $langcode = $this->languageManager->getCurrentLanguage()->getId();
       $location_ids = $this->entityTypeManager->getStorage('node')
         ->getQuery()
         ->accessCheck(FALSE)
         ->condition('type', $type)
         ->condition('status', 1)
+        ->condition('langcode', $langcode)
```

Comment:
what happens with und/en landcodes?
There is a history of language, when node created without translation enabled and then language enabled on a site there will be nodes `und` and `en`

We need to take care about default language before translation enabled on a site @froboy

Reply:
<img width="781" alt="SCR-20240430-magd" src="https://github.com/open-y-subprojects/openy_custom/assets/238201/0445f3da-a477-43ca-975b-dba4784fadf7">

@podarok I tested on a stock install of the distro and it seems like it's impossible to have entities with `und` language codes. On first install of Drupal core, you're required to select a language. Even if you do not enable translations, every entity you create still has a language code from core, completely independent of translation.

<img width="781" alt="SCR-20240430-magd-2" src="https://github.com/open-y-subprojects/openy_custom/assets/238201/336dbb5f-adf4-4031-a3c3-e807d2874003">

So, I think this can move forward.